src/app/team/locationForm/number/LocationNumberView.jsx

```diff
@@ -0,0 +1,154 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Link, withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { deletePhone, getLocation } from '../../../../actions';
+import { getPhoneNumbers } from '../../../../selectors/location';
+
+import Button from '../../../../components/button';
+import ConfirmationOptions from '../../../../components/form/ConfirmationOptions';
+import ConfirmationModal from './ConfirmationModal';
+
+import deleteIcon from './DeleteIcon.svg';
+
+const formatPhoneNumber = phone =>
+  `${phone.number.replace(/\./g, '-')}${
+    phone.extension ? ` ext. ${phone.extension}` : ''
+  }`;
+
+class LocationNumberView extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isEditing: false,
+      showDeleteModal: false,
+    };
+
+    this.toggleEdit = this.toggleEdit.bind(this);
+    this.onConfirmDelete = this.onConfirmDelete.bind(this);
+    this.onCancelDelete = this.onCancelDelete.bind(this);
+    this.deletePhone = this.deletePhone.bind(this);
+  }
+
+  async onConfirmDelete() {
+    const { showDeleteModal: { id } } = this.state;
+
+    await this.props.deletePhone(id);
+
+    this.setState({
+      showDeleteModal: false,
+    }, () => {
+      this.props.fetchResourceData();
+    });
+  }
+
+  onCancelDelete() {
+    this.setState({
+      showDeleteModal: false,
+    });
+  }
+
+  deletePhone(phone) {
+    this.setState({
+      showDeleteModal: phone,
+    });
+  }
+
+  toggleEdit() {
+    this.setState({
+      isEditing: !this.state.isEditing,
+    });
+  }
+
+  render() {
+    const { phones, onConfirm, match } = this.props;
+    const { isEditing, showDeleteModal } = this.state;
+
+    return (
+      <div className="w-100">
+        <div style={{ fontSize: '13px', marginBottom: '1em' }} className="font-weight-bold mt-2">
+          Phone Numbers
+        </div>
+
+        { showDeleteModal &&
+          <ConfirmationModal
+            number={formatPhoneNumber(showDeleteModal)}
+            onCancel={this.onCancelDelete}
+            onConfirm={this.onConfirmDelete}
+          /> }
+
+        <ul className="PhoneNumbers-List">
+          {
+            phones.map(phone => (
+              <li key={`selector-${phone.id}`} className="PhoneNumbers-Element">
+                { isEditing &&
+                  <button
+                    onClick={() => this.deletePhone(phone)}
+                    className="PhoneNumber-Delete"
+                  >
+                    <img src={deleteIcon} alt="Delete" />
```

Comment:
Sorry, my comment was a bit misleading. :(
While we do use the fa-trash icon, it's done through our `Icon` component.

To highlight the value of using that component rather than the raw class names, just a few days ago we actually replaced the font-based icons with svg-based icons, and thanks to the fact we're using a component, we were able to make that change by altering just a single place in the code.